src/metabase/pulse/render/color.clj

```diff
@@ -49,9 +54,23 @@
                       (json/generate-string cols)
                       (json/generate-string viz-settings)))
 
+(defn- parse-number
+  [s]
+  (if (str/includes? s ".")
+    (parse-double s)
+    (parse-long s)))
+
 (defn get-background-color
   "Get the correct color for a cell in a pulse table. Returns color as string suitable for use CSS, e.g. a hex string or
   `rgba()` string. This is intended to be invoked on each cell of every row in the table. See `make-color-selector`
   for more info."
   ^String [color-selector cell-value column-name row-index]
-  (.asString (js/execute-fn color-selector cell-value row-index column-name)))
+  (let [cell-value (cond
+                     (instance? NumericWrapper cell-value)
+                     (-> ^NumericWrapper cell-value
+                         .toString
+                         (str/replace #"," "")
```

Comment:
Will this fail in, e.g., Spanish, where `999 + 1 + ½` is written `1.000,5` instead of `1,000.5`?